### PR TITLE
RDS/JDJ/Rule19-18

### PR DESCRIPTION
**Pull request by JacksonJ-KC**

### CODE REVIEW

File: docs/section19/Rule19-18.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Rule Logic:   
+**Applicability Check 1**     
+Create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_3, HVAC_SYS.SYS_4, HVAC_SYS.SYS_5, HVAC_SYS.SYS_6, HVAC_SYS.SYS_7, HVAC_SYS.SYS_8, HVAC_SYS.SYS_11, HVAC_SYS.SYS_12, HVAC_SYS.SYS_13]`
+- Create dictionary of baseline system types: `baseline_system_types_dict_b = get_baseline_system_types(B_RMD)`  
+- For each hvac system in the B_RMD: `for hvac in B_RMD...HeatingVentilatingAirConditioningSystem:` 
+    - Get the baseline system type: `hvac_sys_type_b = list(baseline_system_types_dict_b.keys())[list(baseline_system_types_dict_b.values()).index(hvac.id)])`
+    - Check if system type is of an applicable system type:  `if any(baseline_system_type_compare(hvac_sys_type_b, target_sys_type, false) for target_system_type in APPLICABLE_SYS_TYPES):`
+    **Rule Assertion:**  
+    - Case 1: Baseline system is one of the applicable system types; outcome is UNDETERMINED: `outcome = UNDETERMINED and raise_message "For <insert hvac.id> perform manual check that the bhp used to model the baseline system fan power aligns with the calculation procedure in Table G3.1.2.9."`
```
Comment (claperle) [inline docs/section19/Rule19-18.md:29]: Should this be indented since it is the then for the if statement?

Comment (claperle) [inline docs/section19/Rule19-18.md:29]: Suggested message:
"For <insert hvac.id> perform manual check that fan motor efficiency established per Table G3.9.1 used for determining  baseline system fan power was based on the next motor size greater than the bhp using a totally enclosed fan cooled motor at 1800 rpm."`

Comment (claperle) [inline docs/section19/Rule19-18.md:29]: There was a previous RDS that covered this. Was it decided to just change this to a non primary rule?

**Comment by JacksonJ-KC** [comment]
(https://github.com/pnnl/ruleset-checking-tool/pull/797) Christina had written the RDS associated with this rule, with much more detail than the Applicability test that I have written here. Do we want to incorporate the previous work?

**Pull request closed without merging**